src/gov-action-contracts/AIPs/MiscAIP/UpdateGasChargeAction.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.16;
+
+import "@arbitrum/nitro-contracts/src/precompiles/ArbOwner.sol";
+import "@arbitrum/nitro-contracts/src/precompiles/ArbGasInfo.sol";
+
+/// @notice Update fixed cost of posting a batch from 100000 to 240000 L1 gas units, which more accurately reflects actual cost.
+/// Also properly disable amortized cost cap (should be 0, not max(int64))
```

Comment:
if we expect it to always be these values, they should be hardcoded as immutables and checked against the prev and new values. 
otherwise this is a bit misleading

```suggestion
/// @notice Update fixed cost of posting a batch, which more accurately reflects actual cost.
/// Also properly disable amortized cost cap (should be 0, not max(int64))
```